docs/static/releasenotes.asciidoc

```diff
@@ -27,6 +28,152 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-7-0]]
+=== Logstash 7.7.0 Release Notes
+
+==== New features and improvements
+
+===== Improving Stack communication - Monitoring
+
+As the {stack} continues to evolve, improvements across stack components become
+necessary to improve performance, internal communication or usability of our
+products. {ls} is no different, as we continuously strive to improve its
+interaction with the rest of the stack. As such, this release brings
+improvements to how Logstash sends monitoring data (from its Internal
+Collection) to {es}.
+
+Previous versions of {es} received monitoring data in a Production cluster to
+then forward it (internally) to the dedicated Monitoring cluster. Newer versions
+improved the process by allowing the data to be sent directly to the Monitoring
+cluster instead, in a different data format. Following those changes, Logstash
+has now added support to send the monitoring data directly to the Monitoring
+cluster, with the caveat that the Production cluster only has to retrieve the
+`cluster_uuid`.
+https://github.com/elastic/logstash/pull/11106[#11106],  
+https://github.com/elastic/logstash/pull/11640[#11640],
+https://github.com/elastic/logstash/pull/11641[#11641]  
+
```

Comment:
About `cluster_uuid` Rob also fixed a problem with hardcoded pipeline name issue: #11775 PR: #11777

Reply:
I think its ok not to include since that Bugfix was just an issue that appeared due to the changes above.